src/tests/suite/daos_epoch_io.c

```diff
@@ -1559,15 +1559,32 @@ epoch_io_setup(void **state)
 	struct epoch_io_args	*eio_arg;
 	char			*tmp_str;
 	int			 rc;
+	char			str[37];
+
+	daos_prop_t *prop;
+
+	prop = daos_prop_alloc(1);
+	assert_non_null(prop);
+	prop->dpp_entries[0].dpe_type = DAOS_PROP_CO_EC_CELL_SZ;
+	prop->dpp_entries[0].dpe_val = 1 << 15;
 
 	obj_setup(state);
 	arg = *state;
+	rc = daos_cont_create(arg->pool.poh, &uuid, prop, NULL);
+	daos_prop_free(prop);
+	assert_success(rc);
+
+	uuid_unparse(uuid, str);
+	rc = daos_cont_open(arg->pool.poh, str, DAOS_COO_RW,
+			    &coh, NULL, NULL);
+	assert_success(rc);
```

Comment:
here you create another extra container and don't use obj_setup() created container, and use global uuid/coh, not a big problem in test code. 
Can be refined by change test_setup() (in calling obj_setup), adding a container property there similar as dt_csum_type.